src/utils.js

```diff
@@ -0,0 +1,18 @@
+const sanitize = function(text) {
+    return [...text].filter(function(char) {
+      const unicodeCode = char.codePointAt(0);
+
+      return !zeroLengthCharacterCodes.includes(unicodeCode);
+    }).join("");
+}
+
+//https://stackoverflow.com/a/18455088/6591929
+const copyTextToClipboard = function (text) {
+  var copyFrom = document.createElement("textarea");
+  copyFrom.textContent = text;
+  var body = document.getElementsByTagName('body')[0];

```

Comment:
Could you do `const body = document.querySelector('body')` to avoid having to access in index?

Reply:
or maybe even just `document.body`?

Reply:
Yes, of course ! I  thought in this changes but I don't know why I didn't, thanks.
I prefer the `querySelector` but in this case `document.body` is better.